### PR TITLE
[Multimedia.MediaTool] Add APIs for rotation and flip

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add APIs for rotation and flip. 
Rotation and flip are in same enum at native fw. But, Rotation and Flips enums are already used in Multimedia so we use it.

### API Changes ###

 - TCSACR-174
   ( It's related with Native ACR-1214.)

Added:
 - Rotation { get; set; } // Property
 - Flip { get; set; } // Property

### CODE REVIEW

File: src/Tizen.Multimedia/MediaTool/MediaPacket.cs [hunk 1]
```diff
@@ -252,6 +252,79 @@ public bool IsEncoded
             }
         }
 
+        /// <summary>
+        /// Gets or sets the rotation value of the current packet.
+        /// </summary>
+        /// <exception cref="ArgumentException">The specified value to set is invalid.</exception>
+        /// <exception cref="ObjectDisposedException">The MediaPacket has already been disposed of.</exception>
+        /// <since_tizen> 5 </since_tizen>
```
Comment (kqjy777) [inline src/Tizen.Multimedia/MediaTool/MediaPacket.cs]: it needs to add InvalidOperationException for ValidateNotLocked.

File: src/Tizen.Multimedia/MediaTool/MediaPacket.cs [hunk 2]
```diff
@@ -252,6 +252,79 @@ public bool IsEncoded
...
+
+        /// <summary>
+        /// Gets or sets the flip value of the current packet.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Flips.None"/> will be ignored in set case. It's not supported in Native FW.
+        /// </remarks>
+        /// <exception cref="ArgumentException">The specified value to set is invalid.</exception>
+        /// <exception cref="ObjectDisposedException">The MediaPacket has already been disposed of.</exception>
+        /// <since_tizen> 5 </since_tizen>
```
Comment (kqjy777) [inline src/Tizen.Multimedia/MediaTool/MediaPacket.cs]: it needs to add InvalidOperationException for ValidateNotLocked.

File: src/Tizen.Multimedia/MediaTool/MediaPacket.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia/MediaTool/MediaPacket.cs]: Done

File: src/Tizen.Multimedia/MediaTool/MediaPacket.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia/MediaTool/MediaPacket.cs]: Done

**Pull request merged**